EasyLearn/app/views/login/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<head>
+<%= stylesheet_link_tag "login", "data-turbo-track": "reload" %>
+</head>
+<body>
+	<section class="vh-100 gradient-custom">
+		<div class="container h-100">
+			<div class="row d-flex justify-content-center align-items-center h-100">
+				<div class="col-12 col-md-8 col-lg-6 col-xl-5">
+					<div class="card bg-dark text-white" style="border-radius: 1rem;">
+						<div class="card-body p-5 text-center">
+							<div class="mb-md-5 pb-3">
+								<h2 >Easy Learn</h2>
+								<%= image_tag("easylearn-icon.png", :alt => "rss feed") %>
+								<div class="text-divider">Login</div>
+								<form>
+									<div class="mb-3">
+										<label for="exampleInputEmail1" class="d-flex justify-content-start">Email</label>
+										<input type="email" class="form-control" id="exampleInputEmail1">
+									</div>
+									<div class="mb-3">
+										<label for="exampleInputPassword1" class="d-flex justify-content-start">Password</label>
+										<input type="password" class="form-control dark" id="exampleInputPassword1">
+									</div>
+									<p class="mb-0">I forgot my <a href="#" class="fw-bold">password</a></p>
+									<button type="submit" class="btn btn-dark"><span>Login</span></button>
+								</form>
+							</div>
+							<hr>
+							<p class="mb-0">Or create a new account <a href="#" class="fw-bold">here</a></p>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</section>
+</body>
```

Comment:
It is not necessary to place the <body> tag.

Add a blank line.

Check the indentation of your files.